src/View/ViewableData.php

```diff
@@ -676,6 +681,12 @@ public function getViewerTemplates($suffix = '')
      */
     public function Me()
     {
+        if (is_a($this, ArrayData::class)) {
+            $keys = array_keys($this->toMap());
+            if (count($keys) === 1 && $keys[0] == '_index') {
+                return $this->_index;
+            }
+        }
```

Comment:
This works fine if you are explicitly only using `$Me`, but fails if I return `['string 1', 'string 2]` from a controller into an array and use this template:
```ss
<% loop $MyIterator %>$Plain<% end_loop %>
```
I'd expect the `$Plain` method from the `DBText` that's wrapping the string to be called, but it will instead try to call it on the `ArrayData` I believe.